web-ng/src/app/components/layer-list/layer-list.component.html

```diff
@@ -22,4 +22,10 @@
   <div *ngFor="let entry of layers">
     <ground-layer-list-item [layer]="entry"> </ground-layer-list-item>
   </div>
+  <div class="add-layer">
+    <button mat-raised-button class="add-layer-btn">
+      <mat-icon class="add-layer-text">add</mat-icon>
+      Add Layer
```

Comment:
"layer" => "Layer" (lowercase)

Reply:
I see what happened- I reversed my comment, very sorry! I meant to say the text should be "Add layer", with "layer" being lowercase. Could you kindly adjust?

Reply:
Please replace button text "Add Layer" with "Add layer" as per previous comment

Reply:
I got it, I will make the changes as you have suggested. Thank you! 😊